### ISSUE TITLE
ambiguous  with labels mapping

### ISSUE THREAD

**Issue by SAIVENKATARAJU** [ibm-aur-nlp/PubLayNet#42]
Hi,
I am currently fine tuning layout parser on my custom dataset. I am using **pubLayNet/faster_rcnn_R_50_FPN_3x** as my base model but according to this model output label set is something like this. `{0: "Text", 1: "Title", 2: "List", 3: "Table", 4: "Figure"}`. but in my original PDF I just want to use `"Title",  "Section", "Paragraph", "ListItem", PageNumber""Table"`. my question is : what should be the order of the label mapping. Also, with use of pre-trained model its pretty much detecting tables in customdata and i Just dont want to ruin it. can you please suggest me how should I Move along?.

**Comment by opyate** [comment]
See [this doc](https://github.com/ibm-aur-nlp/PubLayNet/tree/master/pre-trained-models#convert-pre-trained-model-for-fine-tuning-on-another-target-dataset) - it has instructions for custom label mappings.
